functests/common.sh

```diff
@@ -136,16 +136,17 @@ function add_drives() {
     url=$(minikube service --namespace=directpv-min-io admin-service --url)
     admin_server=${url#"http://"}
 
-    echo -e 'ALL\nALL\nYes\n' | ./kubectl-directpv format --admin-server "${admin_server}"
+    "${DIRECTPV_CLIENT}" discover --admin-server "${admin_server}" --output-file /tmp/drivestoinit.yaml
+    echo -e 'Yes\n\n' | "${DIRECTPV_CLIENT}" init /tmp/drivestoinit.yaml --admin-server "${admin_server}"
```

Comment:
1. Remove two new lines and use single new line with `echo Yes`
2. Generate temporary file instead of `drivestoinit.yaml` and remove the temporary file upon completion.